<!DOCTYPE html>
<title>try-finally-nested-try-catch-within-catch.js</title>
<script charset="UTF-8" src="/ecmascript/harness/assert.js"></script>
<script charset="UTF-8" src="/ecmascript/harness/timer.js"></script>
<script charset="UTF-8" src="/ecmascript/harness/sta.js"></script>
<script charset="UTF-8" src="/ecmascript/harness/cth.js"></script>
<script charset="UTF-8" src="/ecmascript/harness/propertyHelper.js"></script>
<script charset="UTF-8" src="/ecmascript/harness/PromiseHelper.js"></script>
<!-- <script charset="UTF-8" src="/ecmascript/harness/jquery-1.4.2.min.js"></script> -->

<body>
  <script id="test" charset="UTF-8" src="/ecmascript/tests/built-ins/GeneratorPrototype/return/try-finally-nested-try-catch-within-catch.js"></script>
  <script>
    parent.$TEST_COMPLETED();
  </script>
</body>
